parsec/mca/device/device_gpu.c

```diff
@@ -76,7 +80,7 @@ parsec_gpu_check_space_needed(parsec_device_gpu_module_t *gpu_device,
 void parsec_gpu_init_profiling(void)
 {
     if(parsec_gpu_profiling_initiated == 0) {
-        parsec_profiling_add_dictionary_keyword("cuda", "fill:#66ff66",
```

Comment:
Each accelerator should generate their own events, instead of generating generic-named events.

Reply:
`device->name` will contain a string of the form "cuda(2)" where 2 is the dev_index.

Reply:
The device name is translated into the stream naming, but these events being unique they will be unique across all profiling streams, making them matcheable across execution domains. That make sense when we are talking about threads, but for devices it might be a stretch.